packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-node-julia.vue

```diff
@@ -231,17 +233,32 @@ const runCalibrate = async () => {
 	}
 };
 
+const handlingProgress = (message: string) => {
+	const parsedMessage = JSON.parse(message);
+	console.log(parsedMessage);
+};
+
 const getStatus = async (simulationId: string) => {
 	showSpinner.value = true;
 	if (!simulationId) return;
 
 	const runIds = [simulationId];
+
+	// open a connection for each run id and handle the messages
+	runIds.forEach((id) => {
+		eventSourceManager.openConnection(id, `/simulations/${id}/sciml-intermediate-results`);
+		eventSourceManager.setMessageHandler(id, handlingProgress);
+	});
+
 	poller
 		.setInterval(3000)
 		.setThreshold(300)
 		.setPollAction(async () => simulationPollAction(runIds, props.node, progress, emit));
 	const pollerResults = await poller.start();
```

Comment:
Do we still need this if you are implementing the SSE endpoints?

Reply:
This is necessary in order to know when to close the `eventSourceManager`. See below

Reply:
We are using SSE/MQ to communicate `intermediate events`, that is a separate concept than `progress`  - the fact we have been calling it "progress" or "handleProgress" is not technically correct, and I think further exacerbates the the confusions we have here.

Reply:
Yep, so this needs to be renamed to avoid confusion.
This comes back to my notifications system we need to setup to improve UX.

Reply:
Wait sorry, so what is the suggestion here? Should I be making any additional changes to the code here?

Reply:
Rename `handleProgress` to `handleIntermediate` or something similar ..., the poller is polling for progress, but the existing handleProgress is not related to progress, so this was causing some confusion.

Reply:
done!